examples/workflow/authoring/src/activity-sequence.ts

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { DaprWorkflowClient, WorkflowActivityContext, WorkflowContext, WorkflowRuntime, TWorkflow } from "@dapr/dapr";
+
+async function start() {
+  // Update the gRPC client and worker to use a local address and port
+  const daprHost = "localhost";
+  const daprPort = "50001";
+  const workflowClient = new DaprWorkflowClient({
+    daprHost,
+    daprPort,
+  });
+  const workflowRuntime = new WorkflowRuntime({
+    daprHost,
+    daprPort,
```

Comment:
Will it be necessary to pass these args here? If not passed, will `WorkflowRuntime` be created via DAPR_RUNTIME_HOST and DAPR_GRPC_PORT env settings?

Reply:
Yes, so if the they are not passed here, we check the env vars at https://github.com/dapr/js-sdk/blob/6fb65443fbb38a528a30f7289c41a20479b29511/src/workflow/internal/index.ts#L51-L52 to build the workflow runtime.